core/services/fluxmonitor/flux_monitor.go

```diff
@@ -686,12 +693,10 @@ func (p *PollingDeviationChecker) createJobRun(nextPrice decimal.Decimal, nextRo
 	return err
 }
 
-var dec0 = decimal.NewFromInt(0)
-
 // OutsideDeviation checks whether the next price is outside the threshold.
-func OutsideDeviation(curPrice, nextPrice decimal.Decimal, threshold float64) bool {
-	if curPrice.Equal(dec0) {
-		logger.Infow("Current price is 0, deviation automatically met", "currentPrice", dec0)
+func OutsideDeviation(nextAnswer decimal.Decimal, threshold float64) bool {
+	if curAnswer.IsZero() {
```

Comment:
What am I missing here... I can't see where this variable comes from?

Reply:
nothing missing, this was a bad update

Reply:
Local changes were not included in this branch and written over, getting to it soon.